package.json

```diff
@@ -23,8 +23,8 @@
         "@mojaloop/central-services-database": "3.1.0-snapshot",
         "@mojaloop/central-services-error-handling": "0.5.1",
```

Comment:
Please update version to 5.2.0 and run npm install to update package-lock.json

Reply:
done